### PR TITLE
runescape-client: Added Animation runelite api file. Also renamed "skin" to "skeleton".

### PR DESCRIPTION
**Pull request by Noodleeater**
This pull request exposes some methods for getting animation data. Personally, i am using it to export animation data for use in other applications.

### CODE REVIEW

**Review by swazrgb — CHANGES_REQUESTED** [review]

File: runelite-api/src/main/java/net/runelite/api/Animation.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package net.runelite.api;
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/Animation.java:1]: license header

File: runelite-api/src/main/java/net/runelite/api/DualNode.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package net.runelite.api;
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/DualNode.java:1]: license header

File: runelite-api/src/main/java/net/runelite/api/DualNode.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+public interface DualNode extends Node
+{
+	void unlinkDual();
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/DualNode.java:5]: this method is unimplemented, need to `@Import` it in rs-api

File: runelite-api/src/main/java/net/runelite/api/Frames.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package net.runelite.api;
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/Frames.java:1]: license header

File: runelite-api/src/main/java/net/runelite/api/Skeleton.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package net.runelite.api;
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/Skeleton.java:1]: license header

**Review by ThatGamerBlue — CHANGES_REQUESTED** [review]

**Review by ThatGamerBlue — APPROVED** [review]

**Pull request merged**